react/week3/todo-app/src/App.css

```
@@ -0,0 +1,40 @@
+.App {
+  text-align: center;
+}
+.main-Container {
+  margin: 10%;
+  background-color: rgb(207, 89, 89);
+  border: 2px solid;
+  border-radius: 10px;
+  padding: 20px;
+  color: white;
+  text-align: center;
+}
+
+.add-button,
+.delete-button,
+.edit-button,
+.update-button {
+  margin: 5px;
+  background: green;
+  color: white;
+  border-radius: 5px;
+}
+.delete-button {
+  background: red;
+}
+.todo {
+  margin: 5px;
+}
+ul,
+li {
+  list-style: none;
+  color: indigo;
+  font-weight: bold;
+  text-align: center;
+}
+.header-Container {
```

```suggestion
.header-container {
```
kebab-case (dash-case) convention for classes names